lib/rewrite/rewrite-expr-grammar.ym

```diff
@@ -136,6 +140,20 @@ rewrite_expr
           }
         ;
 
+rewrite_settag_opts
+        : rewrite_settag_opt rewrite_settag_opts
+        |
+        ;
+
+rewrite_settag_opt
+	: KW_SET_TAG '(' string ')' 
```

Comment:
This is defined outside, so not needed here:
```
	: KW_SET_TAG '(' string ')' 
	{ 
	  log_rewrite_set_tag_new($3, TRUE, configuration); 
	  free($3); 
	}
```
Don't forget to change `|` to `:` before `rewrite_condition_opt`, when removing this rule.

Reply:
okay.